commit/merkleroot/outcome.go

```diff
@@ -138,15 +139,34 @@ func buildReport(
 	sort.Slice(roots, func(i, j int) bool { return roots[i].ChainSel < roots[j].ChainSel })
 
 	sigs := make([]cciptypes.RMNECDSASignature, 0)
-	if q.RMNSignatures == nil {
-		lggr.Warn("RMNSignatures are nil")
-	} else {
+	if q.RMNSignatures != nil { // TODO: should never be nil, error after e2e RMN integration.
 		parsedSigs, err := rmn.NewECDSASigsFromPB(q.RMNSignatures.Signatures)
 		if err != nil {
-			lggr.Errorw("Failed to parse RMN signatures", "error", err)
-		} else {
-			sigs = parsedSigs
+			lggr.Errorw("Failed to parse RMN signatures returning an empty outcome", "error", err)
+			return Outcome{}
```

Comment:
Should this situation return an error to the top-level plugin?

Reply:
The way plugin is designed, it's the same thing, just that plugin would log the error.